tests/resources.py

```diff
@@ -1,16 +1,6 @@
-# Copyright (c) 2014 Google, Inc.
-# Copyright (c) 2015-2016, 2018-2020 Claudiu Popa <pcmanticore@gmail.com>
-# Copyright (c) 2016 Ceridwen <ceridwenv@gmail.com>
-# Copyright (c) 2018 Nick Drozd <nicholasdrozd@gmail.com>
-# Copyright (c) 2019 Ashley Whetter <ashley@awhetter.co.uk>
-# Copyright (c) 2020-2021 hippo91 <guillaume.peillex@gmail.com>
-# Copyright (c) 2020 David Cain <davidjosephcain@gmail.com>
-# Copyright (c) 2021 Pierre Sassoulas <pierre.sassoulas@gmail.com>
-# Copyright (c) 2021 Marc Mueller <30130371+cdce8p@users.noreply.github.com>
-# Copyright (c) 2022 Daniël van Noord <13665637+DanielNoord@users.noreply.github.com>
-
 # Licensed under the LGPL: https://www.gnu.org/licenses/old-licenses/lgpl-2.1.en.html
 # For details: https://github.com/PyCQA/astroid/blob/main/LICENSE
+# Copyright (c) https://github.com/PyCQA/astroid/blob/main/CONTRIBUTORS.txt
```

Comment:
I would suggest we remove the license header from all test files.

Reply:
It send the message that the test files are not copyrighted, no ? And they were originally. How about ``astroid.*`` and ``tests*`` but not custom scripts ?

Reply:
I just wonder what the benefit of having it in test files would be. IMO it's more or less useless. Those files aren't part of the distribution package and mostly boilerplate. In the end, there is still the LICENSE file which applies to all files, regardless if a file has a license header or not.

_Disclaimer: I'm not a lawyer, this is not legal advice._

Reply:
Also not a lawyer 😄 To be frank I also wondered what is the use of having it in any files at all. Apparently the need to do that was removed in march 1989 when the US joined the Berne Convention (before that anything not expressively copyrighted could be stolen !). When pylint was created the remnant of the practice of putting copyright everywhere were still very much alive, probably the reason why it's done that way. That being said, I'd like to be pretty conservative and careful here as I'd prefer to be safe than sorry in this case.  (Fun fact an actual law person in at my job told us developper to do that for internal unpublished code in 2021. Guess no lawyers ever got sacked for asking to put copyright and trademark everywhere)

Reply:
You can't add too much usually 😄 Some projects include the whole license text in every file. Others, like Python, don't add any headers.

In our case it's much more a question what is practical. My argument would be that it's quite unlikely anyone will ever pursue the copyright or GPL license for pylint / astroid code. Much less so for test code. It's already incredibly difficult to do these things for linux, a much bigger project.

Anyway, I guess a few lines more or less don't really hurt.
_To get back to pylint. There we have added header to general test files, but not the individual functional tests. I think we should keep it that way._

Reply:
I agree we should not touch pylint's functional files especially as it would modify all the result txt files and do a massive diff for no reason.